.github/labeler.yml

```diff
@@ -1,3 +1,9 @@
+cpp:
```

Comment:
could we rename the lable to "file-cpp" or something? just "cpp" seems like a weird label to have

Reply:
ok